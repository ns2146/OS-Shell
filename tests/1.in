path /usr/bin tests
wc tests/10.rc
cat
p1.sh
