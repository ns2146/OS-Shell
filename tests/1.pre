wc tests/10.rc > tests/1.out;
tests/p1.sh >> tests/1.out
