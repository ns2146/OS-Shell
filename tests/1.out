1 1 2 tests/10.rc
test1
test2
test3
test4
